package.json

```diff
@@ -81,7 +81,7 @@
     "tslint": "^5.4.3",
     "tslint-microsoft-contrib": "^5.0.0",
     "tsutils": "^1.9.1",
-    "typescript": "^2.4.2"
+    "typescript": "2.4.2"
```

Comment:
It is yes, typescript 2.5+ introduces suggestions to error messages when it looks like props are misspelt ❤️ .

Typescript doesn't follow semver, so it seemed safer to lock the dev dependency to 2.4.2. anything that work with this _should_ be forwards compatible though :).